src/textacy/extract/triples.py

```diff
@@ -27,9 +27,10 @@
     nsubjpass,
     obj,
     pobj,
-    xcomp,
+    xcomp
 )
 from spacy.tokens import Doc, Span, Token
+import regex as re
```

Comment:
`textacy` doesn't currently have `regex` as a dependency. is it possible to use the stdlib `re` module here instead?

Reply:
probably? I've had some issues using `re` when working with finicky regular expressions, so I kind of use it by default now but I can test it.